cuda_python/docs/source/conf.py

```diff
@@ -87,7 +87,7 @@
 .. _cuda.core: {CUDA_PYTHON_DOMAIN}/cuda-core/latest
 .. _cuda.bindings: {CUDA_PYTHON_DOMAIN}/cuda-bindings/latest
 .. _cuda.pathfinder: {CUDA_PYTHON_DOMAIN}/cuda-pathfinder/latest
-.. _cuda.cccl.cooperative: https://nvidia.github.io/cccl/python/cooperative
-.. _cuda.cccl.parallel: https://nvidia.github.io/cccl/python/parallel
+.. _cuda.cccl.cooperative: https://nvidia.github.io/cccl/python/coop
+.. _cuda.cccl.parallel: https://nvidia.github.io/cccl/python/compute
```

Comment:
Since the package names have changed, could you reflect it to necessary places?
```suggestion
.. _cuda.coop: https://nvidia.github.io/cccl/python/coop
.. _cuda.compute: https://nvidia.github.io/cccl/python/compute
```